src/Application/Users/Models/UserPublicViewModel.cs

```diff
@@ -12,4 +15,10 @@ public record UserPublicViewModel : IMapFrom<User>
     public string Name { get; init; } = string.Empty;
     public Uri? Avatar { get; init; }
     public Region? Region { get; init; }
+    public ClanPublicViewModel? Clan { get; init; } = new();
```

Comment:
Don't initialize the clan with an empty clan. If the user doesn't have a clan it should be null.